agents/unix/conf/base/conf_netconsole.c

```diff
@@ -107,6 +107,83 @@ static netconsole_targets     targets;
 
 static bool netconsole_was_loaded = true;
 
+static te_errno
+get_remote_mac(const struct sockaddr *local_ip,
+               const char *ifname,
+               const struct sockaddr *remote_ip,
+               struct sockaddr *remote_mac)
+{
+    int           s = -1;
+    struct arpreq remote_hwaddr_req;
+    int           rc;
+    int           tmp_err;
```

Comment:
I'd say `te_errno rc;` and instead of `tmp_err = errno; ... return te_rc_os2te(tmp_err)` use `rc = TE_OS_RC(TE_TA_UNIX, errno); ... return rc;`